.eslintignore

```diff
@@ -3,4 +3,14 @@ bakedSite/**
 coverage/**
 itsJustJavascript/**
 devtools/**
-knexfile.ts
\ No newline at end of file
+jest.config.js
```

Comment:
We won't need these since in line 1 we have `**/*.js`.

I tried to get rid of that just now, but there is some other weirdness going on with the `tsconfig.eslint.json` file so I couldn't quite get that to work just now.

But I think the `!wordpress/web/app/plugins/owid/**/*.js` will still work nonetheless.